diff -rup lohit-bengali-2.4.3/Lohit-Bengali.sfd lohit-bengali-2.4.3_mod/Lohit-Bengali.sfd
--- lohit-bengali-2.4.3/Lohit-Bengali.sfd	2009-09-09 15:18:33.000000000 +0530
+++ lohit-bengali-2.4.3_mod/Lohit-Bengali.sfd	2011-02-04 10:27:22.961686320 +0530
@@ -3,7 +3,7 @@ FontName: Lohit-Bengali
 FullName: Lohit Bengali
 FamilyName: Lohit Bengali
 Weight: Book
-Copyright:   Copyright (c) 2003, Automatic Control Equipments, Pune, INDIA.\n  Copyright 2006 Modular Infotech Pvt Ltd.\n  Copyright 2009 Red Hat, Inc.\n\n  Licensed under the GNU General Public License, version 2 (see file COPYING) along with the following exception:\n\n  As a special exception, if you create a document which uses this font, and embed this font or unaltered portions of this font into the document, this font does not by itself cause the resulting document to be covered by the GNU General Public License. This exception does not however invalidate any other reasons why the document might be covered by the GNU General Public License. If you modify this font, you may extend this exception to your version of the font, but you are not obligated to do so. If you do not wish to do so, delete this exception statement from your version.
+Copyright: Copyright (c) 2003, Automatic Control Equipments, Pune, INDIA.\n  Copyright 2006 Modular Infotech Pvt Ltd.\n  Copyright 2009 Red Hat, Inc.\n\n  Licensed under the GNU General Public License, version 2 (see file COPYING) along with the following exception:\n\n  As a special exception, if you create a document which uses this font, and embed this font or unaltered portions of this font into the document, this font does not by itself cause the resulting document to be covered by the GNU General Public License. This exception does not however invalidate any other reasons why the document might be covered by the GNU General Public License. If you modify this font, you may extend this exception to your version of the font, but you are not obligated to do so. If you do not wish to do so, delete this exception statement from your version.
 Version: 2.4.3
 ItalicAngle: 0
 UnderlinePosition: -300
@@ -13,13 +13,14 @@ Descent: 154
 LayerCount: 2
 Layer: 0 1 "Back"  1
 Layer: 1 1 "Fore"  0
+NeedsXUIDChange: 1
 XUID: [1021 648 100124046 5181693]
 FSType: 0
 OS2Version: 1
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1163146539
-ModificationTime: 1250058071
+ModificationTime: 1296795337
 PfmFamily: 17
 TTFWeight: 400
 TTFWidth: 5
@@ -363,9 +364,9 @@ NameList: Adobe Glyph List
 DisplaySize: -36
 AntiAlias: 1
 FitToEm: 1
-WinInfo: 2368 37 13
+WinInfo: 37 37 13
 AnchorClass2: "Anchor-0"  "'abvm' Above Base Mark in Bengali lookup 1" "Anchor-1"  "'abvm' Above Base Mark in Bengali lookup 1" "Anchor-2"  "'blwm' Below Base Mark in Bengali lookup 0" 
-BeginChars: 65869 485
+BeginChars: 65869 486
 
 StartChar: .notdef
 Encoding: 65536 -1 0
@@ -47548,5 +47549,38 @@ SplineSet
 EndSplineSet
 Ligature2: "'pres' Pre Base Substitutions in Bengali lookup 7" u09B2_u09CD.half u09A7
 EndChar
+
+StartChar: uni20B9
+Encoding: 8377 8377 485
+Width: 464
+VWidth: 0
+Flags: HW
+LayerCount: 2
+Fore
+SplineSet
+268.85 375.438 m 0,0,1
+ 269.601 382.197 269.601 382.197 269.601 385.952 c 1,2,-1
+ 44.3076 385.952 l 1,3,-1
+ 63.833 433.264 l 1,4,-1
+ 259.839 433.264 l 1,5,6
+ 255.332 442.275 255.332 442.275 248.573 449.785 c 0,7,8
+ 210.273 491.089 210.273 491.089 123.911 491.089 c 2,9,-1
+ 44.3076 491.089 l 1,10,-1
+ 63.833 538.4 l 1,11,-1
+ 437.819 538.4 l 1,12,-1
+ 418.294 491.089 l 1,13,-1
+ 275.608 491.089 l 1,14,15
+ 304.896 473.816 304.896 473.816 315.41 433.264 c 1,16,-1
+ 437.819 433.264 l 1,17,-1
+ 418.294 385.952 l 1,18,-1
+ 322.169 385.952 l 1,19,20
+ 322.169 269.26 322.169 269.26 128.417 255.282 c 1,21,-1
+ 380.745 -0.0498047 l 1,22,-1
+ 305.647 -0.0498047 l 1,23,-1
+ 52.5684 253.029 l 1,24,-1
+ 52.5684 297.337 l 1,25,26
+ 259.757 295.835 259.757 295.835 268.85 375.438 c 0,0,1
+EndSplineSet
+EndChar
 EndChars
 EndSplineFont
